tests/soil_crop_tests/field_tests/test_field.py

```diff
@@ -78,15 +92,45 @@ def test_manage_field() -> None:
     field._check_crop_harvest_schedule = MagicMock()
     field._remove_dead_crops = MagicMock()
     field._reset_crop_field_coverage_fractions = MagicMock()
+    field._execute_manure_application = MagicMock()
```

Comment:
```suggestion
    execute_manure_application = mocker.patch.object(field, "_execute_manure_application")
```
Mypy is really doesn't like assigning things to a method, even though it works fine.

Reply:
Yea, there are a bunch of these in the test file, wasn't sure if this was the PR to make those changes but I can certainly start with this one!

Reply:
I think that would be better! Consistency is great, but if someone sees the correct way in the code I think they will be more inclined to follow it and fix more of the mocks if they're working on the tests